spec.emu

```diff
@@ -446,16 +455,17 @@ contributors: Google, Ecma International
       </p>
 
       <p>
-        The value of the [[RelevantExtensionKeys]] internal slot is &laquo;  &raquo;.
+        The value of the [[RelevantExtensionKeys]] internal slot is &laquo; `"ca"` &raquo;.
       </p>
 
       <p>
         The value of the [[LocaleData]] internal slot is implementation defined within the constraints described in <emu-xref href="#sec-internal-slots"></emu-xref> and the following additional constraints:
       </p>
 
       <ul>
-        <li>[[LocaleData]].[[&lt;_locale_&gt;]] must have a [[types]] field for all locale values _locale_.  The value of this field must be a Record, which must have fields with the names of the six display name types: `"language"`, `"region"`, `"script"`, `"currency"`, `"weekday"`, `"month"`, `"quarter"`, `"dayPeriod"`, and `"dateTimeField"`.</li>
-        <li>The value of each of those fields in turn must be a Records which must have fields with the names of the three display name styles: `"narrow"`, `"short"`, and `"long"`.</li>
+        <li>[[LocaleData]].[[&lt;_locale_&gt;]] must have a [[types]] field for all locale values _locale_.  The value of this field must be a Record, which must have fields with the names of the nine display name types: `"language"`, `"region"`, `"script"`, `"currency"`, `"weekday"`, `"month"`, `"quarter"`, `"dayPeriod"`, and `"dateTimeField"`.</li>
```

Comment:
```suggestion
        <li>[[LocaleData]].[[&lt;_locale_&gt;]] must have a [[types]] field for all locale values _locale_.  The value of this field must be a Record, which must have fields with the names of one of the valid display name types: `"language"`, `"region"`, `"script"`, `"currency"`, `"weekday"`, `"month"`, `"quarter"`, `"dayPeriod"`, and `"dateTimeField"`.</li>
```
to avoid having to keep the count u to date in the future?